src/components/ProjectRequestForm/FormForProfit.jsx

```diff
@@ -0,0 +1,157 @@
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
+
+export default function FormForProfit({ setFormState, loadPreviousForm, loadNextForm }) {
+  return (
+    <Formik
+      initialValues={{
+        businessPlan: "",
+        linkToDocs: "",
+        systemDefinition: "",
+        CommunityOrProfit: "",
+        isFunded: "",
+      }}
+      validationSchema={Yup.object().shape({
+        businessPlan: Yup.string().required("Required field"),
+        linkToDocs: Yup.string()
+          .url("Invalid url (e.g: http://example.com)")
+          .required("Required field"),
+        systemDefinition: Yup.string().required("Required field"),
+        CommunityOrProfit: Yup.string().required("Required field"),
+        isFunded: Yup.string().required("Required field"),
+      })}
+      onSubmit={(values) => {
+        setFormState((prev) => {
+          return { ...prev, ...values };
+        });
+        loadNextForm("confirmForm")
+      }}
+    >
+      <Form className={styles.requestForm}>
+        <div className={styles.inputField}>
+          <label>Is there a long-term business plan?</label>
+          <Field
+            id="haveBusinessPlan"
+            type="radio"
+            name="businessPlan"
+            value="yes"
+          />
+          <label className={styles.labelRadio} htmlFor="haveBusinessPlan">
+            Yes
+          </label>
+
+          <Field
+            id="noBusinessPlan"
+            type="radio"
+            name="businessPlan"
+            value="no"
+          />
+          <label className={styles.labelRadio} htmlFor="noBusinessPlan">
+            No
+          </label>
+          <ErrorMessage
+            name="businessPlan"
+            render={(msg) => <span className={styles.formError}>{msg}</span>}
+          />
+        </div>
+
+        <div className={styles.inputField}>
+          <label>
+            Is the system defined properly? (provide link to slides)
+          </label>
+          <Field
+            id="systemDefined"
+            type="radio"
+            name="systemDefinition"
+            value="yes"
+          />
+          <label className={styles.labelRadio} htmlFor="systemDefined">
+            Yes
+          </label>
+
+          <Field
+            id="systemNotDefined"
+            type="radio"
+            name="systemDefinition"
+            value="no"
+          />
+          <label className={styles.labelRadio} htmlFor="systemNotDefined">
+            No
+          </label>
+          <ErrorMessage
+            name="systemDefinition"
+            render={(msg) => <span className={styles.formError}>{msg}</span>}
+          />
+
+          <label className={styles.addFileLabel} htmlFor="systemDefinitionFile">
+            Select a file:
+          </label>
+          <input
+            type="file"
+            id="systemDefinitionFile"
+            name="systemDefinitionFile"
+          ></input>
```

Comment:
File should be provided via external link.

Reply:
ok. changed